src/Microsoft.IdentityModel.Tokens/CryptoProviderFactory.cs

```diff
@@ -103,7 +103,7 @@ public CryptoProviderFactory(CryptoProviderFactory other)
         /// <summary>
         /// Gets the <see cref="CryptoProviderCache"/>
         /// </summary>
-        public CryptoProviderCache CryptoProviderCache { get; } = new InMemoryCryptoProviderCache();
+        internal CryptoProviderCache CryptoProviderCache { get; set; } = new InMemoryCryptoProviderCache();
```

Comment:
Looks like a breaking change. Let me know if ApiCompat is not catching this as we recently downgraded ApiCompat version.

Reply:
ApiCompat does catch this! I've fixed this so that only the set is internal.